### PR TITLE
Completed es6-destructuring.

### PR DESCRIPTION
**Pull request by bookaneers**
es6-destructuring | Fernando Figueiredo
Screen Shot 2021-05-11 at 8 36 08 PM

### CODE REVIEW

File: es6-destructuring/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+];
+
+const [book3, book4, book5] = library;
+
+console.log('book3: ', book3);
+console.log('book4: ', book4);
+console.log('book5: ', book5);
+
+const [fourth] = library;
```
Comment (BrettAlbright) [inline es6-destructuring/main.js]: small update needed here. This is assigning the same object as `book3` above. Destructuring does not change the original array, so using only one value here is selecting the first item of the array.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

**Comment by bookaneers** [comment]
es6-destructuring | Fernando Figueiredo
Screen Shot 2021-05-12 at 4 14 38 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**